doc/import_production_data.md

```diff
@@ -0,0 +1,23 @@
+# Import Production Data
+
+The content_performance_manager database is no longer imported by default.
```

Comment:
This is sentence has no context and only makes sense with pre-existing knowledge. I think it would be important to start this documentation with a link to the import production data guide in the tech docs.